### PR TITLE
add fancy travelnet (textures by @El-Naso)

### PR DESCRIPTION
**Pull request by BuckarooBanzay**
Replaces https://github.com/pandorabox-io/pandorabox-textures/pull/3

## TODO

* [ ] proper inv image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,4 @@ Exceptions:
 
 * `textures/travelnet_top.png` [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/) https://github.com/minetest/minetest_game (`default_steel_block.png`)
 * `textures/travelnet_bottom.png` [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/) https://github.com/minetest/minetest_game (`default_clay.png`)
+* `textures/travelnet_fancy*` [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/) provided by @El-Naso
```
Comment (Panquesito7) [inline README.md:39]:
```suggestion
* `textures/travelnet_fancy_*.png` [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/) provided by @El-Naso
```

File: travelnet.lua [hunk 2]
```diff
@@ -38,3 +38,16 @@ travelnet.register_travelnet_box({
...
+travelnet.register_travelnet_box({
+	nodename = "travelnet:travelnet_fancy",
+	tiles = {
+		"travelnet_fancy_front.png",
+		"travelnet_fancy_back.png",
+		"travelnet_fancy_side.png",
+		"travelnet_top.png",
+		"travelnet_bottom.png",
+	},
+	inventory_image = "travelnet_inv_base.png", --TODO: proper inv
```
Comment (Panquesito7) [inline travelnet.lua:51]:
```suggestion
	inventory_image = "travelnet_inv_base.png", -- TODO: proper inv
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
There are [**merge conflicts**](https://github.com/mt-mods/travelnet/pull/15/conflicts). Could you please fix the error? If you don't have time available, I could work on it if wanted. Thanks. 🙂

**Comment by BuckarooBanzay** [comment]
> There are [**merge conflicts**](https://github.com/mt-mods/travelnet/pull/15/conflicts). Could you please fix the error? If you don't have time available, I could work on it if wanted. Thanks. slightly_smiling_face

I initially just dumped this here because i didn't have an idea on how to fit these textures into the existing mod.
A new box perhaps with a "fancy" recipe? Not sure but i'm open for ideas ;)

**Comment by OgelGames** [comment]
Seeing as there is now an API to register travelnet nodes, I think it would make more sense for it to be a separate mod.

https://github.com/mt-mods/travelnet/blob/master/doc/api.md#fully-custom-nodes

**Comment by wsor4035** [comment]
closed by https://github.com/mt-mods/fancy_travelnet

**Pull request closed without merging**